deploy/kubernetes/latest/controller.yaml

```diff
@@ -0,0 +1,233 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: csi-controller-sa
+  namespace: kube-system
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: external-provisioner-runner-role
```

Comment:
NIT: 
```suggestion
  name: external-provisioner-role
```

Reply:
These are coming from: https://github.com/kubernetes-csi/external-provisioner/blob/master/deploy/kubernetes/rbac.yaml

I don't really liking the naming either and if you are ok with changing it I can do that.

Reply:
Yeah, we can decide whatever we like, in this case shorter name feel better to me.